package.json

```diff
@@ -7,14 +7,20 @@
   "license": "MIT",
   "devDependencies": {
     "@shopify/argo-admin-cli": "latest",
+    "@shopify/eslint-plugin": "^37.0.1",
     "@types/yargs": "^15.0.4",
+    "eslint": "^7.7.0",
     "inquirer": "^7.1.0",
+    "prettier": "^2.1.0",
     "ts-node": "^8.9.1",
     "typescript": "^3.0.0",
     "yargs": "^15.3.1"
   },
   "scripts": {
-    "generate": "ts-node -O \"{\\\"esModuleInterop\\\": true}\" ./scripts/generate"
+    "check": "prettier --check \"**/scripts/**/*.{ts,tsx}\"",
+    "format": "prettier --write \"**/scripts/**/*.{ts,tsx}\"",
+    "generate": "ts-node -O \"{\\\"esModuleInterop\\\": true}\" ./scripts/generate",
+    "lint": "eslint . --ext .ts,.tsx --format codeframe"
```

Comment:
I don't think we're picking up the template files

Reply:
Right. I could add `.template` here. I think there's one of two ways to go here

1. We rename to .ts, .tsx, .js, and let it pick it up that way.
1. I think we named it `.template` because there used to be search-and-replace keys on our template files. If we aim to have search-and-replace again in the future, the linting will bug out and we should not include them as part of our linting.

Reply:
@alanthai Sorry, I thought I sent this comment but it was still in draft.

I'm leaning towards option 1 since there's no need to do a search and replace anymore. We might as well simplify until there's a need for it. Thoughts?

Reply:
Yeah I'm happy to go this way. In that case let me rename all the template files.